internal/gitutil/git.go

```diff
@@ -69,7 +70,27 @@ func EnsureUpdated(uri, destinationPath string) error {
 	return updateAndCleanUntracked(destinationPath)
 }
 
-func exec(pwd string, args ...string) error {
+// ListModifiedFiles will fetch origin and list modified files
+func ListModifiedFiles(uri, destinationPath string) ([]string, error) {
```

Comment:
Let's not touch `gitinfo` if we can.

Instead, let's do this:

1. parse index into []pluginInfo
2. update()
3. parse index into []pluginInfo
4. print diff (new and updated plugins), if any

Reply:
The idea to use git for this job is on me. I think it can be helpful because `git diff` can report added/changed/deleted file names.
However, just working with sorted lists is probably the simplest solution of all and does not have to rely on git's output format. I'd be ok either way.